sci-libs/onnx/onnx-1.17.0.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Open standard for machine learning interoperability"
+DESCRIPTION="Open Neural Network Exchange (ONNX) is an open ecosystem that empowers AI developers \
+to choose the right tools as their project evolves. ONNX provides an open source format for AI \
+models, both deep learning and traditional ML. It defines an extensible computation graph model, \
+as well as definitions of built-in operators and standard data types. Currently we focus on the \
+capabilities needed for inferencing (scoring)."
+HOMEPAGE="https://github.com/onnx/onnx"
+COPYRIGHT="204 ONNX Project Contributors"
```

Comment:
```suggestion
COPYRIGHT="2024 ONNX Project Contributors"
```

Reply:
Thanks, this one and onnxruntime have been bugging me for a few days now, parking it and will step away for a bit on that. :)